rook-api/src/main/java/com/github/shyiko/rook/api/event/RowsMutationReplicationEvent.java

```diff
@@ -44,4 +46,8 @@ public String getTable() {
     public T getRows() {
         return rows;
     }
+
+    public long getHostId() {
```

Comment:
@igrunskiy change to getServerId() (to be consistent with MySQLReplicationStream) and we can merge it.